src/components/chat-item/chat-item-card.ts

```diff
@@ -299,6 +301,101 @@ export class ChatItemCard {
       this.card?.render.insertChild('beforeend', this.fileTreeWrapper.render);
     }
 
+    /**
+     * Generate file tree if available
+     */
+    if (this.informationCardWrapper != null) {
```

Comment:
I've thought about it several times, but if we want to use it in places other than the chat-item-card, it would be handy to have this component as a separate component. 

Let's move it to a different file and just call the component here (like the `ChatItemButtons`). 
Together with the `status` block.